### DISCUSSION TITLE
Using a target rate sensor to activate devices

### DISCUSSION THREAD

**Discussion by stephencrowther** [BottlecapDave/HomeAssistant-OctopusEnergy#1059] [Q&A]
I know this is probably the most simplest form of automation in HA, but I just want to check something.

Do I have to create two automations to do this - one to switch device on and one to switch device off or is there a cleverer way to create an automation that kind of just matches the off state of the target rate sensor to the off state of a switch or device and vice versa - basically so it just follows whatever state the target rate sensor does?

thanks

**Comment by BottlecapDave** [comment]
You don't have to, but depending on what you're doing you might find it easier to read/maintain. 

If you want it as a single automation, you can use a trigger with a list of to fields https://www.home-assistant.io/docs/automation/trigger/#examples and then you can have an if else in your action which checks the state of the input entity and act accordingly https://www.home-assistant.io/docs/scripts/#if-then
